mvp-spec/operator-api.md

```diff
@@ -136,66 +210,81 @@ For details on how to calculate or verify signatures, see [signatures.md](signat
 - if `paf_preferences` cookie exists, return its values
 - [_on REST version only_] attempt to create a temporary, short-life, "test 3PC" cookie
 
-#### REST read: `GET /v1/id-prefs`
+#### REST read: `GET /v1/ids-prefs`
 
 | Message  | Format                                                  |
 |----------|---------------------------------------------------------|
-| Request  | [get-id-prefs-request](model/get-id-prefs-request.md)   |
-| Response | [get-id-prefs-response](model/get-id-prefs-response.md) |
+| Request  | [get-ids-prefs-request](model/get-ids-prefs-request.md)   |
+| Response | [get-ids-prefs-response](model/get-ids-prefs-response.md) |
 
-##### Example
+<details>
+<summary>Full example</summary>
+
+- the following request is built:
+
+<!--partial-begin { "files": [ "getIdsPrefsRequest.json" ], "block": "json" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+  "sender": "cmp.com",
+  "receiver": "operator.pafdemo.com",
+  "timestamp": 1643041140,
+  "signature": "lytE4Nel9kwdqqG+Nv6esS5qJqyWIjXiWBbp8DdihM8LK53+dO8djpcNulUT9GLrqCyUCU8mlIrHVggkDGRqIA=="
+}
+```
+<!--partial-end-->
 
-- request
-<!-- The query string below is generated with taking the request-advertiserA.json file, removing body, and encoding it as query string:
-npx encode-query-string -nd `cat request-advertiserA.json | npx json -e 'this.body = undefined' -o json-0`
--->
+  - and transformed into a parameter of the query string to form the URL to call:
 
+<!--partial-begin { "files": [ "getIdsPrefsRequest.http" ], "block": "http" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 ```http
-GET /v1/id-prefs/read?sender=advertiserA.com&timestamp=1639057962145&signature=message_signature_xyz1234
+GET https://operator.pafdemo.com/v1/ids-prefs?paf=%7B%22sender%22%3A%22cmp.com%22%2C%22receiver%22%3A%22operator.pafdemo.com%22%2C%22timestamp%22%3A1643041140%2C%22signature%22%3A%22lytE4Nel9kwdqqG%2BNv6esS5qJqyWIjXiWBbp8DdihM8LK53%2BdO8djpcNulUT9GLrqCyUCU8mlIrHVggkDGRqIA%3D%3D%22%7D
```

Comment:
If you're using the HTTP format you can't put the full url after the get - it confuses the syntax highlighter.
Maybe add the Host: header

Reply:
Thanks, fixed